matrix.json

```diff
@@ -0,0 +1,77 @@
+{
+    "uuid": "4BA68789-FF92-4649-B7D8-65BCE3966589",
+    "version": "2.0",
+    "organisation": {
+        "name": "Shift CryptoAG",
+        "web": "https://shiftcrypto.ch/",
+        "email": "support@shiftcrypto.ch",
+        "location": "Switzerland",
+        "established": "2018-01-23"
+    },
+    "wallet": {
+        "name": "Bitbox02",
+        "web": "https://shiftcrypto.ch/bitbox02/",
+        "socials": [
+            {
+                "type": "twitter",
+                "slug": "shiftcryptoHQ"
+            },
+            {
+                "type": "medium",
+                "slug": "shiftcrypto"
+            },
+            {
+                "type": "linkedin",
+                "slug": "shift-crypto"
+            }
+        ],
+        "version": ""
+    },
+    "features": {
+        "currencies": [
+            {
+                "type": "crypto",
+                "ticker": "BTC,LTC,ETH,BAT"
+            },
+            {
+              "type": "stablecoin",
+              "ticker": "USDT"
+            }
+        ],
+        "wallet_types": ["on-chain"],
+        "security": ["touch-id","fake-wallet"],
```

Comment:
> These seem to be missing:
> 
> - pin-on-wallet
> - encrypted-password
> - 2fa
> 
> Remove `touch-id`. While BitBox02 does have touch-sensing controls, it doesn't authenticate. According to [the glossary](https://walletmatrix.app/glossary#featuresecurity) it is supposed to authenticate.

Not sure if encrypted password fits tbh since there's no passwords stored on the bb02

Reply:
Oh right. I read too fast. Was actually thinking about encrypted seed. :+1:

Reply:
what's a fake wallet :joy:

Reply:
A "Fake Wallet" is one that only contains a fraction of a person's holdings in order to "palm-off" a would-be attacker that these are your only funds.